R/bundle.R

```diff
@@ -126,6 +126,13 @@ createAppManifest <- function(appDir,
                               verbose = FALSE,
                               quiet = FALSE) {
 
+  if (is.null(image)) {
```

Comment:
The final image value is inferred "late", but that lets both `deployApp()` and `writeManifest()` avoid duplication.